# cloud is still imported despite tensorflow bulit with nogcp. See:
# https://github.com/tensorflow/tensorflow/issues/29617
diff -ru tensorflow-1.13.1.orig/tensorflow/contrib/__init__.py tensorflow-1.13.1/tensorflow/contrib/__init__.py
--- tensorflow-1.13.1.orig/tensorflow/contrib/__init__.py	2019-02-25 21:37:04.000000000 +0100
+++ tensorflow-1.13.1/tensorflow/contrib/__init__.py	2021-07-02 10:19:17.276767786 +0200
@@ -26,8 +26,6 @@
 from tensorflow.contrib import batching
 from tensorflow.contrib import bayesflow
 from tensorflow.contrib import checkpoint
-if os.name != "nt":
-  from tensorflow.contrib import cloud
 from tensorflow.contrib import cluster_resolver
 from tensorflow.contrib import coder
 from tensorflow.contrib import compiler
